fuel-tests/Cargo.toml

```
@@ -0,0 +1,29 @@
+[package]
+name = "fuel-tests"
+version = "0.1.0"
```

Since it's not being published
```suggestion
version = "0.0.0"
```